docker-compose.yml

```diff
@@ -0,0 +1,74 @@
+version: '3.7'
+
+volumes:
+  redis-data:
+  export-node-modules:
+
+services:
+  minio:
+    image: minio/minio:RELEASE.2023-06-09T07-32-12Z
+    restart: unless-stopped
+    ports:
+      - 9000:9000
+      - 9090:9090
+    volumes:
+      - ~/minio-data-litefarm:/data
+    environment:
+      - MINIO_ROOT_USER=myminioadmin
+      - MINIO_ROOT_PASSWORD=myminioadmin
+      - CONSOLE_ADDRESS=9090
+    command: server /data --console-address ":9090"
+
+  create_minio_bucket:
+    image: minio/mc:RELEASE.2023-06-06T13-48-56Z
+    depends_on:
+      - minio
+    entrypoint: >
+      /bin/sh -c "
+      /usr/bin/mc alias set myminio http://minio:9000 myminioadmin myminioadmin;
+      /usr/bin/mc mb myminio/minio-dev;
+      /usr/bin/mc anonymous set public myminio/minio-dev;"
+
+  redis:
+    image: redis:7.0-alpine
+    restart: unless-stopped
+    command: redis-server --requirepass test --loglevel warning
+    volumes: # Persisted volume required by bull
+      - redis-data:/data
+    ports:
+      - 6379:6379
+
+  export_server:
+    image: kathyavini/node-awscli-lf:chromium
```

Comment:
@iperdomo it became too much of a hassle to use discordant a Node version so I have just made the image myself. I don't have a Docker Pro account so I'd love to know how/where to set this up better; on the free account you can't connect GitHub.

Docker Hub Link: https://hub.docker.com/r/kathyavini/node-awscli-lf